src/test/java/edu/cornell/kfs/paymentworks/util/PaymentWorksUtilTest.java

```diff
@@ -0,0 +1,85 @@
+package edu.cornell.kfs.paymentworks.util;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksCustomFieldDTO;
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksCustomFieldsDTO;
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksFieldChangeDTO;
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksFieldChangesDTO;
+
+public class PaymentWorksUtilTest {
+	
+	PaymentWorksUtil paymentWorksUtil;
+	private static final String PAYMENT_WORKS_FIELD_NAME = "field name";
+	private static final String PAYMENT_WORKS_FIELD_FROM_VALUE = "old value";
+	private static final String PAYMENT_WORKS_FIELD_TO_VALUE = "new value";
+
+	@Before
+	public void setUp() throws Exception {
+		paymentWorksUtil = new PaymentWorksUtil();
+		Logger.getLogger(PaymentWorksUtil.class).setLevel(Level.DEBUG);
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		paymentWorksUtil = null;
+	}
+
+	@Test
+	public void testPojoToJsonString() {
```

Comment:
This isn't a huge deal, but I mentioned it to Daniela in another review so I'll mention it to you here. Since we can use annotations to indicate test methods now, the idiom I'm seeing is to name your test method after the method it is testing along with possibly some context and then dropping the "test" prefix. So this method name would be pojoToJsonString. If you were testing it with invalid input, it might be pojoToJsonStringInvalidInput for example.